AppKit/CPTextView/CPLayoutManager.j

```diff
@@ -1125,6 +1125,20 @@ var _objectsInRange = function(aList, aRange)
     return nil;
 #endif
 }
+- (id)createDOMElementWithImage:(CPImage)anImage
+{
+#if PLATFORM(DOM)
+    var style,
+        img = document.createElement("img");
+
+    img.oncontextmenu = img.onmousedown = img.onselectstart = _oncontextmenuhandler;
+    // fixme: config the img accoringly
+
```

Comment:
```
var style,
        img = document.createElement("img"),
        span = document.createElement("span");

    style = span.style;
    style.position = "absolute";
    style.visibility = "visible";
    style.padding = "0px";
    style.margin = "0px";
    style.whiteSpace = "pre";
    style.backgroundColor = "transparent";

    img.oncontextmenu = img.onmousedown = img.onselectstart = _oncontextmenuhandler;
    // fixme: config the img accoringly

    img.src = [anImage filename];
    img.height = [anImage size].height;
    img.width = [anImage size].width;

    span.innerHTML = "<img src='"+[anImage filename]+"' height='"+[anImage size].height+"' width='"+[anImage size].width+"'>";

    //span.appendChild(img);

    return span;
```

This is not yet the code that it should be, but at the moment I tried something to make things work ...